source/localizable/tutorial/routes-and-templates.md

```diff
@@ -141,7 +141,7 @@ Here we will use the `{{link-to}}` helper in our code to link between routes:
     By building a property rental site, we can simultaneously imagine traveling
     AND building Ember applications.
   </p>
-  {{#link-to 'index' class="button"}}
+  {{#link-to 'contact' class="button"}}
```

Comment:
I just did that because it says "When we look at our about page at
http://localhost:4200/about, we now have a working link to our contact page."
underneath that. I don't mean to be nitpicky haha. Have a good one.

On Thu, Jun 9, 2016 at 4:41 PM, Todd Jordan <notifications@github.com>
wrote:

> In source/localizable/tutorial/routes-and-templates.md
> <https://github.com/emberjs/guides/pull/1479#discussion_r66540978>:
>
> > @@ -141,7 +141,7 @@ Here we will use the `{{link-to}}` helper in our code to link between routes:
> >      By building a property rental site, we can simultaneously imagine traveling
> >      AND building Ember applications.
> >    </p>
> > -  {{#link-to 'index' class="button"}}
> > +  {{#link-to 'contact' class="button"}}
>
> I think we do in fact mean to link to index here. We want the user to "Get
> started" picking out a rental.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/emberjs/guides/pull/1479/files/df0fe509844d9e41f93d6068a55984511480835f#r66540978>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMTgqIJpKTi8O_zLvb6Pqaqas8RYYUGDks5qKKSYgaJpZM4IyeaN>
> .
>



-- 
*W. Dane Iliff*
Junior Engineer

T 310 765 1600 • M 310 633 0716

5766 West Jefferson Blvd • Los Angeles, CA 90016